src/store/elementalChat.js

```diff
@@ -104,14 +104,27 @@ export default {
     channels: [],
     currentChannelId: null,
     stats: {},
-    statsLoading: false
+    statsLoading: false,
+    agentHandle: '',
```

Comment:
This should probably start out `null` given that when we reset it that's what we set it to
```suggestion
    agentHandle: null,
```